docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.14.3-alpine
+WORKDIR /
+ENV CGO_ENABLED=0
+COPY . /
+ARG TARGETOS
+ARG TARGETARCH
+RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build
+
+EXPOSE 8080
+CMD ["./sparagn-media-service"]
```

Comment:
I am not sure that is a good practice to run the actual service during the docker build rather then in docker-compose after the build, but as long as it is working  and it is not locking the CircleCI machine is fine.

Reply:
there are different ways for this. Many of them use RUN, CMD or ENTRYPOINT like here https://github.com/afdolriski/golang-docker/blob/master/Dockerfile . In docker-compose we had to do that in the other project coz we had in many services to run with dependencies which is not the case here. I think is fine for now, maybe we'll come back with another PR on this

Reply:
ok, that is fine for now